tests/gh-actions/install_knative.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 set -euo pipefail
 echo "Installing KNative ..."
-set +e
-kustomize build common/knative/knative-serving/overlays/gateways | kubectl apply -f -
-set -e
-kustomize build common/knative/knative-serving/overlays/gateways | kubectl apply -f -
-kustomize build common/knative/knative-eventing/base | kubectl apply -f -
+
+kustomize build common/knative/knative-serving/base | kubectl apply -f -
```

Comment:
I was assuming that something was wrong and hence the repetition. Thanks for the confirmation.
The test I think will fail even otherwise because of change in knative 1.8, will add a fix for it later today or tomorrow.